release notes/v0.4.0.md

```diff
@@ -0,0 +1,105 @@
+xk6-browser v0.4.0 is here! :tada:
+
+In this minor release, we implemented a large portion of the Locator API, fixed critical bugs, and made some improvements and code refactoring!
+
+
+## New features
+
+- Implemented `page.locator(selector)` ([#310](https://github.com/grafana/xk6-browser/pull/310))
+
+  This will create and return a new page locator given a selector with strict mode on (only select a single element that matches and if there are multiple matches an error is returned). Check below for what can be done with the locator.
+
+- Added `locator.click([options])` which allows an unique element to be clicked ([#318](https://github.com/grafana/xk6-browser/pull/318))
```

Comment:
Happy to do that. @inancgumus could you please supply the body of the description and the small example? I can then remove the details and just have a more concise list of the features that were implemented for this new API.

Reply:
Sure. After you finish all the requested changes, I'll fix this by sending another commit to this PR.

Reply:
@inancgumus I did some restructuring of the Locator API section. It was faster to do it myself than point things out in comments. So let me know how it looks.

Reply:
Makes sense! Thanks for the restructuring, it looks better :thumbsup:  I'd like to change this (`Hold modifier key (e.g. Shift) now works when working with keyboard.down and keyboard.up. (#325)`) to the conventional `Fixed ...` format if you have stopped working on the release notes?

Reply:
> So it might be confusing, but proper sentences should have periods. List items and headers don't need to. :)

Good call 👍 I was indecisive about it. It's better [this way]([a1afa7d](https://github.com/grafana/xk6-browser/pull/418/commits/a1afa7d07efce27ebb889f71b46c02f8e114eab7)).